tools/kwiver_tool_runner.cxx

```diff
@@ -134,43 +313,11 @@ void tool_runner_usage( VITAL_UNUSED applet_context_t ctxt,
   }
 }
 
-// ----------------------------------------------------------------------------
-/**
- * This function handles the "help" operation. If there is an arg
- * after the help, then that arg is taken to be the applet name and
- * help is displayed for it.
- *
- * If the only arg is "help", then call the function to generate short
- * help for all known applets.
- */
-void help_applet( const command_line_parser& options,
-                  applet_context_t tool_context,
-                  kwiver::vital::plugin_manager& vpm )
-{
-  if ( options.m_applet_args.size() < 2 )
-  {
-    tool_runner_usage( tool_context, vpm );
-    return;
-  }
-
-  // Create applet based on the name provided
-  applet_factory app_fact;
-  std::string buf = "-- Not Set --";
-  auto fact = app_fact.find_factory( options.m_applet_args[1] );
-  fact->get_attribute( kwiver::vital::plugin_factory::PLUGIN_DESCRIPTION, buf );
-
-  kwiver::tools::kwiver_applet_sptr applet( app_fact.create( options.m_applet_args[1] ) );
-  tool_context->m_applet_name = options.m_applet_args[1];
-  applet->initialize( tool_context.get() );
-  applet->add_command_options();
-
-  // display help text
-  std::cout << applet->m_cmd_options->help();
-}
-
 // ============================================================================
 int main(int argc, char *argv[])
 {
+command_node_sptr cmd_root;
```

Comment:
indentation looks odd here.  Also, could you just declare this later when it is initialized?

Reply:
Done